R/session_check_reauth.R

```diff
@@ -50,7 +50,8 @@ session_check_reauth = function(username, password, session, ...){
 	}else{ #if we have passed nothing, use cached session
```

Comment:
probably, yes. session_check_reauth is confusing to me now, so I am wondering how to deal with it. It probably needs to pass the "ask" stuff we talked about.